pkg/actors/actors.go

```diff
@@ -225,6 +229,18 @@ func newActorsWithClock(opts ActorsOpts, clock clock.WithTicker) (ActorRuntime,
 
 	a.timers.SetExecuteTimerFn(a.executeTimer)
 
+	if opts.Config.SchedulerService != "" {
+		log.Warn("Using scheduler service for reminders.")
```

Comment:
Not sure this should be a warning? Its explicit in configuration

Reply:
Just for our own debugging. I can downgrade to Info.